lib/FileCard/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, PropTypes } from 'react';
+import cx from 'classnames';
+import binSVG from './../../assets/icons/image-trash.svg';
+import commentSVG from './../../assets/icons/image-comment.svg';
+import fullScreenSVG from './../../assets/icons/image-fullscreen.svg';
+import downloadSVG from './../../assets/icons/image-download.svg';
+
+class FileCard extends Component {
+  static propTypes = {
+    image: PropTypes.string,
+    hasComments: PropTypes.bool,
+    filename: PropTypes.string,
+    label: PropTypes.string,
+    type: PropTypes.string,
+  }
+
+  constructor() {
```

Comment:
Yeah, it will (and it does). I'm just sure we'll need a class based component once we actually add more complexity to this component with the upload stuff. But you're right, and I'll follow my own advice regarding premature optimisation 👍